konnectd/assets/identifier/static/js/identifier-container.569688ae.chunk.js

```diff
@@ -0,0 +1,2 @@
ount"})),l.a.createElement(L.a,{gutterBottom:!0},l.a.createElement(A.a,{id:"konnect.goodbye.message.close",defaultMessage:"You can close this window now."})))}),I()(null!==a&&!0===a.state)(function(){return l.a.createElement("div",null,l.a.createElement(L.a,{variant:"h5",component:"h3"},l.a.createElement(A.a,{id:"konnect.goodbye.confirm.headline",defaultMessage:"Hello {displayName}",values:{displayName:a.displayName}})),l.a.createElement(L.a,{variant:"subtitle1",className:n.subHeader},l.a.createElement(A.a,{id:"konnect.goodbye.confirm.subHeader",defaultMessage:"please confirm sign out"})),l.a.createElement(L.a,{gutterBottom:!0},l.a.createElement(A.a,{id:"konnect.goodbye.message.confirm",defaultMessage:"Press the button below, to sign out from your Kopano account now."})),l.a.createElement(N.a,null,l.a.createElement("div",{className:n.wrapper},l.a.createElement(R.a,{color:"secondary",className:n.button,onClick:function(t){return e.logoff(t)}},l.a.createElement(A.a,{id:"konnect.goodbye.signoutButton.label",defaultMessage:"Sign out"})))))}))}},{key:"logoff",value:function(e){var t=this;e.preventDefault(),this.props.dispatch(ue()).then(function(e){var n=t.props.history;e.success&&(t.props.dispatch(le()),n.push("/goodbye"))})}}]),t}(l.a.PureComponent),at=Object(u.b)(function(e){return{hello:e.common.hello}})(Object(p.withStyles)(function(e){return{subHeader:{marginBottom:5*e.spacing.unit},wrapper:{marginTop:5*e.spacing.unit,position:"relative",display:"inline-block"}}})(nt)),rt=function(e){var t=e.component,n=e.hello,a=Object(b.a)(e,["component","hello"]);return l.a.createElement(d.b,Object.assign({},a,{render:function(e){return n?l.a.createElement(t,e):l.a.createElement($e,{target:"/identifier"})}}))};!function(){var e=f("#bg-enhanced.enhanced","background-image",!0),t=f("#bg-enhanced.enhanced::after","background-image",!0),n=[];e&&n.push(new Promise(function(t){var n=new Image;n.onload=function(){t()},n.src=e})),t&&n.push(new Promise(function(e){var n=new Image;n.onload=function(){e()},n.src=t})),Promise.all(n).then(function(){window.document.getElementById("bg-enhanced").className+=" enhanced"})}();var ot=function(e){function t(){return Object(a.a)(this,t),Object(o.a)(this,Object(c.a)(t).apply(this,arguments))}return Object(i.a)(t,e),Object(r.a)(t,[{key:"render",value:function(){var e=this.props,t=e.classes,n=e.hello,a=e.pathPrefix;return l.a.createElement("div",{className:t.root},l.a.createElement(m.a,{basename:a},l.a.createElement(d.d,null,l.a.createElement(rt,{path:"/welcome",exact:!0,component:tt,hello:n}),l.a.createElement(d.b,{path:"/goodbye",exact:!0,component:at}),l.a.createElement(d.b,{path:"/",component:Ze}))))}},{key:"reload",value:function(e){e.preventDefault(),window.location.reload()}}]),t}(s.PureComponent);t.default=Object(u.b)(function(e){var t=e.common;return{hello:t.hello,updateAvailable:t.updateAvailable,pathPrefix:t.pathPrefix}})(Object(p.withStyles)(function(){return{root:{position:"relative",display:"flex",flex:1}}})(ot))}}]);
```

Comment:
Codacy found an issue: [Avoid assignments in operands](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=6202733)